config/common.py

```diff
@@ -116,7 +116,7 @@ def _format_httpjson(record, extras, ignore_keys):
                 '--report-file=$SCRATCH/regression/maintenance/reports/maint_report_{sessionid}.json',
                 '--save-log-files',
                 '-p \'(?!PrgEnv-ce)\'',
-                '--tag=maintenance',
+                '--tag=(maintenance|production)',
```

Comment:
I would keep only the maintenance tag in this mode. It could be that at some point we want to have longer daily tests, that are not suitable for after a maintenance. I prefer we add the maintenance tests to individual apps/tests, as you already did

Reply:
```suggestion
                '--tag=maintenance',
```